project/AppDependencies.scala

```diff
@@ -5,24 +5,23 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     play.sbt.PlayImport.ws,
-    "org.reactivemongo" %% "play2-reactivemongo"            % "0.20.13-play28",
-    "uk.gov.hmrc"       %% "play-frontend-hmrc"             % "3.6.0-play-28",
+    "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"             % "0.73.0",
+    "uk.gov.hmrc"       %% "play-frontend-hmrc"             % "3.32.0-play-28",
     "uk.gov.hmrc"       %% "domain"                         % "8.1.0-play-28",
-    "com.typesafe.play" %% "play-json-joda"                 % "2.9.2",
-    "uk.gov.hmrc"       %% "play-conditional-form-mapping"  % "1.11.0-play-28",
-    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"     % "5.24.0"
+    "com.typesafe.play" %% "play-json-joda"                 % "2.9.3",
+    "uk.gov.hmrc"       %% "play-conditional-form-mapping"  % "1.12.0-play-28",
+    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"     % "7.8.0"
   )
 
   val test: Seq[ModuleID] = Seq(
-    "org.scalatest"               %% "scalatest"                % "3.2.12",
+    "org.scalatest"               %% "scalatest"                % "3.2.14",
     "org.scalatestplus.play"      %% "scalatestplus-play"       % "5.1.0",
     "org.scalatestplus"           %% "scalatestplus-scalacheck" % "3.1.0.0-RC2",
     "wolfendale"                  %% "scalacheck-gen-regexp"    % "0.1.2",
-    "org.jsoup"                   %  "jsoup"                    % "1.15.1",
+    "uk.gov.hmrc.mongo"           %% "hmrc-mongo-test-play-28"  % "0.73.0",
+    "org.jsoup"                   %  "jsoup"                    % "1.15.3",
     "com.typesafe.play"           %% "play-test"                % PlayVersion.current,
-    "org.mockito"                 %  "mockito-all"              % "1.10.19",
-    "org.mockito"                 %  "mockito-core"             % "4.6.1",
-    "org.scalatestplus"           %% "mockito-3-12"             % "3.2.10.0",
+    "org.scalatestplus"           %% "mockito-4-6"              % "3.2.14.0",
     "org.scalacheck"              %% "scalacheck"               % "1.16.0",
     "com.github.tomakehurst"      %  "wiremock-standalone"      % "2.27.2",
     "com.vladsch.flexmark"        %  "flexmark-all"             % "0.62.2"
```

Comment:
all the overrides down in this file need to be deleted forgot to add this last time

Reply:
done